libs/langchain-google-common/src/chat_models.ts

```diff
@@ -58,7 +58,9 @@ export interface ChatGoogleBaseInput<AuthOptions>
   extends BaseChatModelParams,
     GoogleConnectionParams<AuthOptions>,
     GoogleAIModelParams,
-    GoogleAISafetyParams {}
+    GoogleAISafetyParams {
+  modelName?: string;
```

Comment:
Should we add this directly to `GoogleAIModelParams`?

Reply:
Agreed, fixing